### DISCUSSION TITLE
Procedura (workflow) podglądu zmian.

### DISCUSSION THREAD

**Discussion by speplinski** [irdpl/wcag#8] [Organizacja pracy]
Pewnie warto zintegrować automatyczną publikację z pomocą GitHub Pages, używając GitHub Actions i uruchomiać skrypty automatycznie po każdej zmianie w repozytorium. Dzięki temu można będzie na bieżąco podglądać zmiany.

Ewentualnie można wykorzystać narzędzie CI/CD, takie jak Travis CI lub CircleCI by automatycznie wywoływały narzędzie do budowania strony, takie jak Jekyll lub Hugo i wygenerowały strony po zmianie w repozytorium.

**Comment by zwiastunsw** [comment]
Tak to jest teraz zrobione. Wersja przeglądowa jest publikowana automatycznie z pomocą Github Pages, w oparciu o Github Actions.

Pozostańmy przy GithubActions.
